content/docs/1.7.0/v2-data-engine/prerequisites.md

```diff
@@ -47,4 +47,5 @@ SPDK leverages huge pages for enhancing performance and minimizing memory overhe
 
 ### Disk
 
-**Local NVMe disks** are highly recommended for optimal storage performance of volumes using V2 Data Engine.
\ No newline at end of file
+SPDK can use every kind of disk that is supported by Linux, making leverage on Kernel drivers. But SPDK is equipped with a user space NVMe driver which provide zero copy, highly parallel access directly to an SSD from an user space application, so 
+ **Local NVMe disks** are highly recommended for optimal storage performance of volumes using V2 Data Engine.
```

Comment:
```suggestion
SPDK leverages kernel drivers to support every kind of disk that Linux supports. However, SPDK is equipped with a user space NVMe driver that provides zero-copy, highly parallel, direct access to an SSD from a user space application. Because of this, using **local NVMe disks** is highly recommended for enabling V2 volumes to achieve optimal storage performance.
```

Reply:
Much better, thanks.